pontoon/base/static/js/tour.js

```diff
@@ -66,11 +66,14 @@ var update_tour_status = function(step) {
           "to localize various Mozilla project.",
         listeners: {
           beforeStep: function() {
+            // Take the user directly to next step of where he left.
             if(tourStatus != 0)
               Sideshow.gotoStep(tourStatus+1);
           },
           afterStep: function() {
-            update_tour_status(1);
+            if (tourStatus === 0) {
+              updateTourStatus(tourStatus+1);
```

Comment:
I like this idea! However, if the request to save the status to the server fails, then the status number won't get updated, and on subsequent calls, the number will be off. 

For example: 

1 -> OK, status = 1
2 -> FAIL, status = 1
3 -> OK, status = 2

Next time the user comes to the demo page, they will be sent back to page 2 instead of 3. I think the simple way to solve this is to simply pass `tourStatus++` as a parameter, and not use the value returned by the server at all.